docs/.link-check-errors.txt

```diff
@@ -1,4 +1,5 @@
 Broken links found by `yarn docs:link-check` that exist:
 
 - https://mui.com/blog/material-ui-v4-is-out/#premium-themes-store-✨
+- https://mui.com/joy-ui/guides/using-joy-ui-and-material-ui-together
```

Comment:
This should not happen. There are places that link to `https://mui.com/joy-ui/guides/using-joy-ui-and-material-ui-together`, which should be fixed to the new location?

Reply:
Fixed! Thanks for pointing it out!